src/main/scala/com/cloudera/sparkts/DateTimeIndex.scala

```diff
@@ -423,6 +478,54 @@ class HybridDateTimeIndex(
     "hybrid," + dateTimeZone.toString + "," +
       indices.map(_.toString).mkString(";")
   }
+
+  override def millisIterator(): Iterator[Long] = {
+    new Iterator[Long] {
```

Comment:
Would it be possible to just do `indices.map(_.iterator).foldLeft(Iterator[Long]())(_ ++ _)`?